src/ui_utils.c

```diff
@@ -2566,6 +2582,10 @@ void ui_init(void)
 	ui_init_toolbar_widgets();
 	init_document_widgets();
 	create_config_files_menu();
+
+	// after UI is initialized, apply user's custom CSS (if it exists)
+	// to override other CSS styles
+	init_user_style();
```

Comment:
Ah, it's because you call `ui_add_config_file_menu_item()` in the same function, which relies on the menu to exist.